### PR TITLE
002 navigation routes

### PR DESCRIPTION
**Pull request by remi59800**

### CODE REVIEW

**Review by ThomasDCW — APPROVED** [review]

File: frontend/src/App.jsx [hunk 1]
```diff
@@ -1,12 +1,29 @@
-import Home from "@pages/Home";
+import { Routes, Route } from "react-router-dom";
+import Header from "./components/Header";
```
Comment (LuigiMenez) [inline frontend/src/App.jsx:2]: Use absolute please imports

File: frontend/src/pages/Home/index.jsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import Style from "./style";
+
+function Home() {
+  return (
+    <Style>
```
Comment (LuigiMenez) [inline frontend/src/pages/Home/index.jsx:5]: Avoid using <Style> as a tag in your components, customise them.

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Use absolute please imports, and avoid using <Style> as a tag in your components, customise them.

**Review by remi59800 — COMMENTED** [review]
fix absolut import and name balise Style

**Pull request closed without merging**